ui/tool_tip_summaries.h

```diff
@@ -0,0 +1,467 @@
+/*
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+#include <unordered_map>
+#include <string>
+
+// Generated summaries for Vulkan API commands, used for tooltips in the UI.
+const std::unordered_map<std::string, const char*> kVulkanCommandTooltipSummaries = {
```

Comment:
Please don't instantiate variable in header file.
Also global non-constexpr variables are bad idea.

See https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables or go/totw/110

file.h
```
const std::unordered_map<std::string_view, std::string_view>& GetVulkanCommandTooltipSummaries();
```
file.cpp
```
const std::unordered_map<std::string_view, std::string_view>& GetVulkanCommandTooltipSummaries() {
  static const auto* impl  =
    new std::unordered_map<std::string_view, std::string_view>{
    // ...
  };
  return *impl;
}
```